v3-docs/docs/.vitepress/config.mts

```diff
@@ -40,6 +40,14 @@ export default defineConfig({
                 text: "Meteor + Vue + vue-meteor-tracker",
                 link: "/tutorials/vue/meteorjs3-vue3-vue-meteor-tracker",
               },
+              {
+                text: "Meteor.js 3 + Blaze",
```

Comment:
probably we will have a conflict when merge it, we should have only the Svelte item here